libcontainer/cgroups/utils.go

```diff
@@ -269,7 +269,6 @@ func RemovePaths(paths map[string]string) (err error) {
 			}
 		}
 		if len(paths) == 0 {
-			paths = make(map[string]string)
```

Comment:
This needs to be kept https://github.com/opencontainers/runc/pull/2628

Reply:
@AkihiroSuda PTAL